src/referencePackages/buildReferencePackages.proj

```diff
@@ -29,6 +39,18 @@
     <Message Importance="High" Text="Copy DummyFiles Complete."/>
   </Target>
 
+  <Target Name="BuildTargettingPacks" BeforeTargets="Build" Condition="'@(TargetingPacksSrc)' != ''">
+    <Message Importance="High" Text="[$([System.DateTime]::Now.ToString('HH:mm:ss.ff'))] Building TargetingPacks assemblies." />
+    <MakeDir Directories= "$(TargetingPacksDLLPath) " />
+    <Exec Command="$(IlasmToolPathSB)ilasm %(TargetingPacksSrc.Identity) -dll -quiet -nologo -output=$(TargetingPacksDLLPath)%(TargetingPacksSrc.Filename).dll" />
+    <Message Importance="High" Text="[$([System.DateTime]::Now.ToString('HH:mm:ss.ff'))] Done building TargetingPacks assemblies." />
+    <Message Importance="High" Text="[$([System.DateTime]::Now.ToString('HH:mm:ss.ff'))] Packing targeting Packs only ." />
+    <Exec Command="$__DOTNET_CMD restore %(TargetingPacksProjects.Identity)"/>
+    <Exec Command="$__DOTNET_CMD pack --no-build --output $(ArtifactsRefPkgDir) /nowarn:NU5125 %(TargetingPacksProjects.Identity)"/>
+    <Message Importance="High" Text="Exec Command=$__DOTNET_CMD pack --no-build --output $(ArtifactsRefPkgDir) /nowarn:NU5125 %(TargetingPacksProjects.Identity)" />
+    <Message Importance="High" Text="[$([System.DateTime]::Now.ToString('HH:mm:ss.ff'))] Packing targeting Packs Done" />
+  </Target>
```

Comment:
This needs to clean up -

- [ ] Add dlls to artifacts

- [ ] Remove Debug Messages